modules/ROOT/pages/debugging-the-runtime-manager-agent.adoc

```diff
@@ -1,15 +1,84 @@
 image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
 image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
 
-You can use the `$MULE_HOME/bin/amc_debug` script to collect information used by Support to troubleshoot the Anypoint Runtime Manager agent installation issues. If you experience issues with the Runtime Manager agent plug-in or with managing applications on Anypoint Platform, run the script and save the output before filing a support ticket.
+Connection issues between your on-premises Mule runtime engine and the Runtime Manager agent might cause discrepancies, such as metrics not appearing in the server monitoring dashboard in Runtime Manager.
+
+== Troubleshoot Connection Issues Between the Agent and Mule
+
+To troubleshoot connection issues:
+
+. Check the JVM you are using.
++
+If you are using the IBM JVM, you must install the truststore manually because this truststore requires different parameters than Oracle JVM or OpenJDK.
++
+See xref:servers-create.adoc#ibm-jvm[IBM JVM].
+. Ensure you are using the latest agent version.
++
+.. Check the agent version.
++
+In the Runtime Manager navigation menu, select *Servers* and then click the server name.
+The agent version appears on the *Dashboard* page:
++
+.The arrow shows the Runtime Manager agent version on the server *Dashboard* page.
+image::rtm-agent-version.png[Runtime Manager agent version on the server Dashboard page]
+.. Check the xref:release-notes::runtime-manager-agent/runtime-manager-agent-release-notes.adoc[Runtime Manager Agent Release Notes] to determine the latest agent version compatible with your Mule version.
+.. Update the agent, if necessary.
++
+See xref:installing-and-configuring-runtime-manager-agent.adoc#update-the-agent[Update the Agent].
+. Verify that the license and certificate are not expired.
++
+.. In the Runtime Manager navigation menu, select *Servers* and then click the server name.
+The expiration dates appear on the *Dashboard* page:
++
+.The arrow shows the expiration dates on the server *Dashboard* page.
+image::rtm-server-expirations.png[Expiration dates on the server Dashboard page]
++
+//DASHBOARD SHARED 
+include::partial$cluster-server.adoc[tag=viewServerinClusterOrGroup]
++
+* If the license has expired, see xref:mule-runtime::installing-an-enterprise-license.adoc[Install an Enterprise License].
+* If the certificate has expired, see xref:servers-cert-renewal.adoc[Renew Server Certificates].
+. Verify connectivity with all required hosts.
+.. Ensure that your allowlist includes the required hostnames.
++
+See xref:rtm-agent-whitelists.adoc[Ports, IP Addresses, and Hostnames to Whitelist].
```

Comment:
You mention "allowlist" in line 50, but "whitelist" here. Should they both be "allowlist" per our recent inclusion guidelines? 

It looks like the title of the linked page includes "whitelist", so maybe it's beyond the scope of this PR to consider making a change?

Reply:
The names haven't changed yet. Still scoping.